downloads.markdown

```diff
@@ -6,48 +6,40 @@ isDownloads: true
+### What it is
 
-### Haskell IDEs & other distributions
+<a name="platform"></a>The Haskell Platform is a self-contained, all-in-one installer. After download, you will have everything necessary to build Haskell programs against a core set of useful libraries. It comes in both core versions with tools but no libraries outside of GHC core, or full versions, which include a broader set of globally installed libraries.
 
-In addition to the generic, cross-platform Haskell toolchain described above, there are also easy-to-use, platform-specific distributions and IDEs. The Haskell Wiki contains a [list of the most popular ones](https://wiki.haskell.org/Distributions).
+### What you get
 
-* * *
+*   The [Glasgow Haskell Compiler](https://www.haskell.org/ghc)
+*   The [Cabal build system](https://www.haskell.org/cabal/), which can install new packages, and by default fetches from [Hackage](https://hackage.haskell.org/), the central Haskell package repository.
+*   the [Stack](http://docs.haskellstack.org) tool for developing projects
+*   Support for profiling and code coverage analysis
+*   35 core & widely-used [packages](https://www.haskell.org/platform/contents.html)
 
-## Minimal installers
+### How to get it
 
-### What they are<a name="minimal"></a>
+The Platform is provided as a single installer, and can be downloaded at the links below.
 
-Minimal installers provide centrally the [GHC](https://www.haskell.org/ghc) compiler and the [Cabal](https://www.haskell.org/cabal/) and [Stack](https://github.com/commercialhaskell/stack) tools for installing packages. Some may install further build tools (i.e. for parsing and lexing) as well.
+*   [Linux](http://www.haskell.org/platform/linux.html)
+*   [OS X](http://www.haskell.org/platform/mac.html)
+*   [Windows](http://www.haskell.org/platform/windows.html)
 
-### What you get
+* * *
 
-*   Only the core libraries necessary for each platform are included.
-*   Cabal or Stack must be used to download and install packages after installation.
+### Haskell IDEs & other distributions
 
-### How to get them
+In addition to the generic, cross-platform Haskell toolchain described above, there are also easy-to-use, platform-specific distributions and IDEs. The Haskell Wiki contains a [list of the most popular ones](https://wiki.haskell.org/Distributions).
```

Comment:
The wiki page is outdated, is it not? I don't see any information about modern IDEs. "Haskell for mac" runs ghc-8.6.4, etc.

Reply:
if the wiki page is outdated, people should edit it. its a wiki!

Reply:
How can we ensure that people are consistently keeping the Wiki up to date? This seems difficult, especially since getting a Wiki account is non-trivial.

Reply:
getting a wiki account is trivial. you just email and ask for it. i see a few accounts added each day.

there is no way to ensure the wiki is kept up to date. but there is no way to ensure anything else is kept up to date either! people either do it, or they don't, and then content becomes stale. this is true no matter what.

Reply:
(one caveat: if you tell people nothing, then this never becomes out of date, but then... you have told them nothing.)

Reply:
> if the wiki page is outdated, people should edit it. its a wiki!

It is not a public wiki. I'm trying to find the discussion, but very recently a user reported to have requested an account 3+ months ago and never got any reply.

Reply:
The wiki is public, but simply requires a registration step. The barrier to entry is "send an email". The only alternative is for it to be filled by spam bots. If a user requested an account and didn't get one, their email was lost. I see requests for accounts filled efficiently almost every day.

Reply:
> The only alternative is for it to be filled by spam bots.

There are technical solution to this, such as using a wiki that supports github OAuth. haskell gitlab also uses that model.

Reply:
FTR, this wiki is not as open as you suggest gbaz.

![image](https://user-images.githubusercontent.com/10122937/110583452-9ff14000-8165-11eb-94fb-574c489789df.png)

This prevents correcting/updating the page to keep it in sync with reality.

We should do away with the wiki and let people host their own content and/or combine good, maintained content into the haskell.org website.

Reply:
That is a single protected page (there are a few) on a generally open to edit wiki. The reason that page is protected is by request -- people were concerned that since it contained authoritative information (including a link to donations) that it be specially protected from vandalism. Conversations on an old PR are not a place to have further discussion about plans for the wiki.